app/src/test/java/waiter/HttProtocolTest.java

```diff
@@ -27,4 +27,14 @@ void simpleGetWithBodyResponse() {
         assertTrue(response.contains("HTTP/1.1 200 OK"));
         assertTrue(response.contains("\r\n\r\nHello world"));
     }
+
+    @Test
+    void headSimpleGetResponse() {
+        String fromClient = "HEAD /simple_get HTTP/1.1";
+        HttProtocol httProtocol = new HttProtocol(new RequestParser(), new Router(), new ResponseBuilder());
+
+        String response = httProtocol.serve(fromClient);
+
+        assertTrue(response.contains("HTTP/1.1 200 OK"));
```

Comment:
Is there a way to assert that the response has no body?

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/17/commits/01218f48d001e0c113e1b007fc2d2fe63f0aad2b)